pip/backend.py

```diff
@@ -135,8 +134,14 @@ def _base_setup_args(self, isolate=False):
         ] + list(self.global_options)
 
     def _install_build_reqs(self, reqs, prefix):
+        # Circular import still required
```

Comment:
So I guess in `operations/prepare.py` we can install build requirements into the prefix from `get_518_requirements` and `get_prefix`.